README.md

```diff
@@ -65,6 +65,9 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/a
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
 
+##### This project is Awesome
+designed by Sargon Saadi The Great.
```

Comment:
this is not acceptable

Reply:
Okayn let me think of something funnier.